src/axom/quest/tests/quest_intersection_shaper.cpp

```diff
@@ -0,0 +1,551 @@
+// Copyright (c) 2017-2023, Lawrence Livermore National Security, LLC and
+// other Axom Project Developers. See the top-level LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+/*!
+ * \file
+ * \brief Unit tests for quest's IntersectionShaper class replacement rules.
+ *
+ */
+
+#include "gtest/gtest.h"
+
+#include "axom/config.hpp"
+#include "axom/core.hpp"
+#include "axom/klee.hpp"
+#include "axom/quest.hpp"
+#include "axom/sidre.hpp"
+#include "axom/slic.hpp"
+#include "axom/quest/IntersectionShaper.hpp"
+
+#ifndef AXOM_USE_MFEM
+  #error "Quest's IntersectionShaper tests on mfem meshes require mfem library."
+#endif
+#ifdef AXOM_USE_MPI
+  #include <mpi.h>
+#endif
+#include <cmath>
+#include <string>
+#include <vector>
+
+//#define AXOM_SEQ_INTERSECTION_SHAPER_WORKS
```

Comment:
Aren't the tests disabled when this macro is disabled?

Reply:
Yes, the serial tests were disabled. The logic was that the IntersectionShaper can't really be built in serial. However, that does not apply to the typical RAJA-enabled build which does supply a working SEQ execution policy. So, I renamed the macro here to RUN_AXOM_SEQ_TESTS and left it defined. This way, we can turn it off if we decide they are too slow, which is something I'd probably do during development.